pkg/scanners/helm/parser/parser.go

```diff
@@ -232,7 +232,6 @@ func (p *Parser) writeBuildFiles(tempFs string) error {
 		if err != nil {
 			return err
 		}
-		fmt.Printf("path: %s, rootPath: %s\n", path, p.rootPath)
```

Comment:
I see you're cleaning up after me.....

Reply:
:broom: :broom:

Yeah, test failed as i fixed the rego getting run twice so the number was different, so noticed the output